lib/util.lua

```diff
@@ -0,0 +1,32 @@
+--
+-- SPDX-FileCopyrightText: (c) 2025 jperon <cataclop@hotmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+--- Utility functions.
+-- @module util
```

Comment:
as it's restrict to bin2hex and hex2bin I would name this module something like: encode.{bin2hex, hex2bin} or hexbin.{to,from}.. of something else if you have a better idea ;-)